pytest_fixtures/component/subscription.py

```diff
@@ -1,12 +1,24 @@
 import pytest
 
+from robottelo.cli.subscription import Subscription
+from robottelo.constants import SATELLITE_SUBSCRIPTION_NAME
+
 
 @pytest.fixture(scope='session')
 def clean_rhsm(session_target_sat):
     """removes pre-existing candlepin certs and resets RHSM."""
     session_target_sat.remove_katello_ca()
 
 
+@pytest.fixture(scope='module')
+def default_subscription(module_org_with_manifest):
+    subscription = Subscription.exists(
```

Comment:
I think using `module_default_sat.api.Subscription` and returning an API entity is a better pattern for the fixture, and one that's consistent with the fixture design in robottelo.